specifications/network/messaging-v1.md

```diff
@@ -112,7 +100,9 @@ Pending inbound and outbound messages MAY be reordered and dropped according to
 
 ## Errors
 
-Errors are sent as messages of type `NetworkMessage::Error`, with the `ErrorCode` indicating the type of error. For example, if an `RpcRequest` is received for a `ProtocolId` that was not advertised to a node, we send an error message with the code `ErrorCode::NotSupported(ProtocolId)`.
+Errors are sent as messages of type `NetworkMessage::Error`, with the `ErrorCode` indicating the type of error. For example, if an `RpcRequest` is received for a `ProtocolId` that was not advertised to a node, we send an error message with the code `ErrorCode::NotSupported(1, ProtocolId)`, where 1 represents the index for RpcRequest's in NetworkMessage.
+
+Responding to errors is not requried. A message must be of at least length 2 in order to trigger an error response, otherwise an error would have insufficient data to be meaningful.
```

Comment:
```suggestion
Responding to errors is not requred. A message must be of at least length 2 in order to trigger an error response, otherwise an error would have insufficient data to be meaningful.
```

Reply:
:smh: need to use ispell....